source/rainerscript/constant_strings.rst

```diff
@@ -13,3 +13,36 @@ If in doubt how to properly escape, use the `RainerScript String Escape
 Online
 Tool <http://www.rsyslog.com/rainerscript-constant-string-escaper/>`_.
 
+Type of String Constants
```

Comment:
Would this be "Types of String Constants" instead?

Some other ideas (none of which are blockers):

1. Rename file to `string_constants.rst`
1. Update doc title to `String Constants`
1. Add `Uses` section with most of the info from the first section
1. Rename `Type of String Constants` to `Types`

I've push a branch to my repo that will build shortly for demo purposes.